lib/crypto.js

```diff
@@ -663,10 +663,10 @@ function filterDuplicates(names) {
 
 // Legacy API
 exports.__defineGetter__('createCredentials',
-  internalUtil.deprecate(function() {
-    return require('tls').createSecureContext;
-  }, 'crypto.createCredentials is deprecated. ' +
-     'Use tls.createSecureContext instead.'));
+                         internalUtil.deprecate(function() {
+                           return require('tls').createSecureContext;
+                         }, 'crypto.createCredentials is deprecated. ' +
+                           'Use tls.createSecureContext instead.'));
```

Comment:
@targos Simplifying a bit, this is what the above code boils down to:

```
func1(arg1String,
          arg2Function(function() {
                                   return foo;
                                }, innerArgString));
```

I wonder if the real obstacle to readability is the string concatenation (which makes it look like two string arguments when it's just one).

Also, the final closing parenthesis might make more sense on its own line.

Would removing the concatenation and moving the parenthesis help? Or is the issue something else?